TourMate/TourMate/Frontend/Plan/Views/PlanViews/Components/TimingView.swift

```diff
@@ -8,8 +8,8 @@
 import SwiftUI
 
 struct TimingView: View {
-    @Binding var plan: Plan
-    @State var dateFormatter = DateFormatter()
+    let plan: Plan
+    @State private var dateFormatter = DateFormatter()
```

Comment:
Actually why is the dateformatter a state variable?

Reply:
I wouldn't be able to assign to self otherwise

Reply:
I think u should actually assign the date formatter on init(). Apparently I was reading and they said onAppear might appear too late on some cases